src/config/CemuConfig.h

```diff
@@ -479,9 +479,9 @@ struct CemuConfig
 	// audio
 	sint32 audio_api = 0;
 	sint32 audio_delay = 2;
-	AudioChannels tv_channels = kStereo, pad_channels = kStereo, input_channels = kMono;
-	sint32 tv_volume = 50, pad_volume = 0, input_volume = 50;
-	std::wstring tv_device{ L"default" }, pad_device, input_device;
+	AudioChannels tv_channels = kStereo, pad_channels = kStereo, input_channels = kMono, portal_channels = kMono;
```

Comment:
Should the portal have a setting to adjust the number of channels? The portal always plays mono sound right? This value controls the amount of channels the source is able to have, not the amount of channels on which playback happens.

Reply:
Yeah this is a good point - I will remove this as a setting